src/lib/coda_commands/coda_commands.ml

```diff
@@ -378,20 +378,14 @@ let get_status ~flag t =
           ; consensus_time_best_tip= None } )
   in
   let next_proposal =
-    let str time =
```

Comment:
Cli interface should stay the same I think just for json mode we use the milliseconds (it’s too hard to read as millis as a human)

Reply:
The display is in seconds for plain cli. Next proposal now displays the next_proposal as `consensus_time`